### DISCUSSION TITLE
Can we use 2.3.3 with spring-boot 2.6.3?

### DISCUSSION THREAD

**Discussion by sreekanth-tf** [awspring/spring-cloud-aws#223] [Q&A] [answered]
Can we use version 2.3.3 with spring-boot 2.6.3? If not, can anyone help me to understand what could be the problems? 
I know 2.5.x is supported, but my doubts is as we are planning to move the spring-boot 2.6.3 is it safe to use.

**Comment by maciejwalkowiak** [comment, marked as answer]
Ideally you should wait for Spring Cloud AWS 2.4 (with https://github.com/awspring/spring-cloud-aws/pull/211 merged) - there is a high chance that 2.3.3 will work with Spring Boot 2.6 but this PR aligns all the dependencies. 

2.3.4 and 2.4 will be released in upcoming days.

**Reply by sreekanth-tf** [reply]
Thank you for the update @maciejwalkowiak :+1:
